crates/virtio-queue/src/lib.rs

```diff
@@ -9,6 +9,7 @@
 // Copyright (C) 2020 Alibaba Cloud. All rights reserved.
 //
 // SPDX-License-Identifier: Apache-2.0 AND BSD-3-Clause
+#![feature(stmt_expr_attributes)]
```

Comment:
This allows me to use #[allow(deref_nullptr)] inside the macro definition, but is experimental. I added this to avoid getting warnings everytime offset_of! is used. I could delete this, if you consider it to be too dangerous. In any case, this shows that we should probably look for a rework of the offset_of! macro.